bot_python_sdk/store.py

```diff
@@ -1,76 +1,107 @@
 import json
 import os
+from bot_python_sdk.logger import Logger
 
 _actions_file_path = 'storage/actions.json'
 _last_triggered_file_path = 'storage/last_triggered.json'
 _qr_image_path = 'storage/qr.png'
 _configuration_file_path = 'storage/configuration.json'
 _bot_public_key = 'storage/public.pem'
 
+LOCATION = 'Store'
+
 
 class Store:
+    def __init__(self):
+        pass
+
     @staticmethod
     def set_actions(actions):
-        file = open(_actions_file_path, 'w')
-        file.write(json.dumps(actions))
-        file.close()
+        try:
+            with open(_actions_file_path, 'w') as action_file:
```

Comment:
can you rename it to `actions_file` plural please

Reply:
variable name has been renamed